README.md

```diff
@@ -1 +1,38 @@
-# mongoose-crud
\ No newline at end of file
+# mongoose-crud
+Enable all CRUD(Create, Read, Update, and Delete) endpoints with this express middleware. 
+
+## Features
+- Creates endpoints for `Get` `Post`, `Update`, and `Delete`
+- Enables query strings for `Get`
+
+## Install
+```
+$ npm install mongoose-crud --save
+```
+
+## API
+```javascript
+import Crud from 'mongoose-crud';
+```
```

Comment:
(or) const Crud = require('mongoose-crud')

in some cases you may need to specify default:

const Crud = require('mongoose-crud').default

Reply:
This is because Babel 6 now treats `default` as a named export.  This doesn't mean anything if your using ES6, but if using old javascript, then the `.default` has to be added.  I added this to the Readme.  Thanks for pointing this out

Reply:
May want to look into Webpack for a more elegant solution. It can be configured so that it outputs your library in a way that can be accessed in multiple ways, ES2015 - `include`, CommonJS - `require`, AMD - `require(['moduleName'], ...)` or even a `script` tag in HTML (if desired).

https://webpack.js.org/guides/author-libraries/